config/_default/params.toml

```diff
@@ -12,8 +12,8 @@ docsVersion = "0.1"
 ## Open Graph
 images = ["doks.png"]
 ogLocale = "en_US"
-domainTLD = "doks.netlify.app"
-titleHome = "Doks Theme"
+domainTLD = "wiki.seasalt.ai"
+titleHome = "Seasalt.ai"
```

Comment:
@amy17519 inside this config, could you please LMK:

1. what does L13 `images = ["doks.png"]` do? can you replace it with ours? I can see the example at https://getdoks.org/doks.png but don't understand what it's meant for.
2. I can't find the file on L31 `schemaLogo = "logo-seasalt.png"`

Reply:
I haven't explored all the features beyond the webpage itself. These options are for Open Graph and JSON-LD (https://getdoks.org/docs/reference-guides/seo/#open-graph--twitter-cards). Should be helpful for SEO?! I will update these fields to be related to Seasalt.